src/components/ConnectionsListProfiles.tsx

```diff
@@ -34,6 +34,10 @@ const ConnectionsListProfiles = ({
     (state) => state.profiles?.profiles || {}
   );
 
+  const userFollowing = useAppSelector(
```

Comment:
userImFollowing?  I feel like we can make this a little more descriptive

Reply:
This is a map of all users the current logged in user is following (as opposed to users the display user is following). So `userImFollowing1 is probably not what we want. I guess we could get really verbose. `usersCurrentUserIsFollowing`. Slightly shorter: `followedByCurrentUser`? I kind of like that.